### DISCUSSION TITLE
VIM for 18 B9 S4 P+ B&O - Set to 199 kph, but turns off at 1 kph.

### DISCUSSION THREAD

**Discussion by ddovidenko** [Mr-MIBonk/M.I.B._More-Incredible-Bash#131] [General]
Not sure if this is a bug or limitation of my MMI, other features like map updates seem to work fine. I tried setting the VIM to 199, the MMI rebooted, showed 199 as the current setting, but still turns off when rolling. One thing I noticed was that it was set to 1 kph when I first looked in the vim menu after flashing, but it shows the default as 6 kph in the menu. Seems to turn off right around 1 kph regardless of being set to 199.

**Comment by harman-f** [comment]
There are a few FW versions around there VIM is broken.

This might be one of them - not sure.

Could you please share your LOG file.

**Comment by ddovidenko** [comment]
My MMI Software version is 1320. Do you think I should try to upgrade it? I'll grab logs tomorrow when I have a chance, really appreciate the help!
